Formula/c/certifi.rb

```diff
@@ -15,24 +15,35 @@ class Certifi < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "1f1fc985a1c89bd40c73b17e3dfbf5483cb0417c8d4d12e2be66158a503ab169"
   end
 
-  depends_on "python@3.11" => [:build, :test]
-  depends_on "python@3.12" => [:build, :test]
   depends_on "python@3.13" => [:build, :test]
+  depends_on "python@3.11" => :test # oldest Python to support (not all pythons to avoid `pip install` conflicts)
   depends_on "ca-certificates"
 
   def pythons
-    deps.map(&:to_formula).sort_by(&:version).filter { |f| f.name.start_with?("python@") }
+    deps.filter_map { |dep| dep.to_formula if dep.name.start_with?("python@") }
   end
 
   def install
-    pythons.each do |python|
-      python_exe = python.opt_libexec/"bin/python"
-      system python_exe, "-m", "pip", "install", *std_pip_args(build_isolation: true), "."
+    # Ensure uniform bottles
+    inreplace "README.rst", "/usr/local", HOMEBREW_PREFIX
+
+    build_python = deps.find { |dep| dep.name.start_with?("python@") && dep.build? }
+                       .to_formula
+    python_exe = build_python.opt_libexec/"bin/python"
+    system python_exe, "-m", "pip", "install", *std_pip_args(build_isolation: true), "."
+
+    # Use brewed ca-certificates PEM file instead of the bundled copy
+    site_packages = prefix/Language::Python.site_packages(python_exe)
+    rm(site_packages/"certifi/cacert.pem")
+    (site_packages/"certifi").install_symlink Formula["ca-certificates"].pkgetc/"cert.pem" => "cacert.pem"
+
+    # Create symlinks so the Python bindings can be used with other Python versions
+    oldest_pyver = pythons.min_by(&:version).version.major_minor
+    build_python.versioned_formulae.each do |python|
+      pyver = python.version.major_minor
+      next if pyver < oldest_pyver
 
-      # Use brewed ca-certificates PEM file instead of the bundled copy
-      site_packages = Language::Python.site_packages("python#{python.version.major_minor}")
-      rm prefix/site_packages/"certifi/cacert.pem"
-      (prefix/site_packages/"certifi").install_symlink Formula["ca-certificates"].pkgetc/"cert.pem" => "cacert.pem"
+      (lib/"python#{pyver}/site-packages").install_symlink site_packages.children
```

Comment:
```suggestion
      (lib/"python#{pyver}/site-packages/certifi").install_symlink (site_packages/"certifi").children
```
might be nicer to make sure that the `__pycache__` ends up in the right place? (Unless it doesn't help?)

Reply:
Or, alternatively:
```suggestion
      site_packages.find do |pn|
        next unless pn.file?
        
        target = lib/"python#{pyver}/site-packages"/pn.relative_path_from(site_packages)
        target.parent.install_symlink pn
      end
```

Reply:
> might be nicer to make sure that the `__pycache__` ends up in the right place? (Unless it doesn't help?)

The location shouldn't matter in terms of functionality as it will load them correctly.

Only difference with previous layout is that cache files weren't cleaned between upgrades and installed into `HOMEBREW_PREFIX/lib/...`. However, I would guess the cache file would be invalidated on upgrades anyway so retaining them wouldn't matter.

---

Mostly just changing this is for personal preference of how directory/file structure looks given you can even redirect all cache files to an arbitrary directory via `PYTHONPYCACHEPREFIX`

Reply:
Yea maybe doesn't matter too much. Would still be nicer to fix them all though, but maybe not nice enough to be worth rebuilding stuff and then losing older bottles in the process.